waspc/README.md

```diff
@@ -316,6 +316,21 @@ NOTE: When you run it for the first time it might take a while (~10 minutes) for
 ## Commit message conventions
 We use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) convention when creating commits.
 
+## Branching and merging strategy
+This repo contains both the source code that makes up a Wasp release (under `waspc`), as well as our website containing documentation and blog posts (under `web`). In order to facilitate the development of Wasp code while still allowing for website updates or hotfixes of the current release, we have decided on the following minimal branching strategy.
+
+All Wasp development should be done on feature branches. They form the basis of PRs that will target one of the two following branches:
+
+- `main`: this branch contains all the actively developed new features and corresponding documentation updates. Some of these things may not yet be released, but anything merged into `main` should be in a release-ready state.
+  - This is the default branch to target for any Wasp feature branches.
+- `release`: this branch contains the source code of current/latest Wasp release, as well as the documentation and blog posts currently visible on the website.
+  - When we do a full release, we first backmerge `release` into `main` to pick up any diverging changes (see below). We then merge `main` into `release` to get all the new changes. We can now create a new Wasp release as well as deploy the website from the `release` branch.
```

Comment:
```suggestion
  - When doing a full release, which means making a new release based on what we have currently on `main`, we do the following:
    1. Update `release` branch by merging `main` into it. There might be conflicts, since `release` might have hotfixes and blog posts that `main` doesn't have, but they shouldn't be too hard to fix. Once `release` is updated, you can create a new waspc release from it, as well as deploy the website from it.
    2. To keep `main` up to date with any changes in `release`, you should now also update `main` to reflect the new `release`. There will be no conflicts since we already resolved all of them in the previous step.
```

Reply:
I tried to simplify language a bit, since I found it a bit too technical for me!

And I believe you can then remove the explanation at the bottom, under NOTE.

Reply:
Updated with your suggestion (with one tweak to clarify the how: "2. Update `release` branch to this new `main` by merging `main` into it.") Removed NOTE and references to backmerge. Thanks! 👍🏻